src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -595,28 +594,6 @@ public async Task ConnectionWorkflow_DownloadServiceParameters_Cancelled_AbortsW
             AssertServiceDisconnectCalled();
         }
 
-        [TestMethod]
-        public void IsSonarQubePluginSupported_WhenUnsupportedVersionOfVSAndCSharpPluginEquals70_ReturnsFalseAndWriteExpectedText()
-        {
-            TestPluginSupport(
-                expectedResult: false,
-                vsVersion: "14.0",
```

Comment:
The deleted tests were all testing what happened in V2015 RTM, so they are no longer needed.